datadog/tests/data_source_datadog_cloud_workload_security_agent_rules_test.go

```diff
@@ -0,0 +1,68 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-datadog/datadog"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+const tfAgentRulesSource = "data.datadog_cloud_workload_security_agent_rules.acceptance_test"
+
+func TestAccDatadogCloudWorkloadSecurityAgentRulesDatasource(t *testing.T) {
```

Comment:
Could you update this test so that a new CloudWorkloadAgentRule is first created and then read using the datasource. Here is an example: https://github.com/DataDog/terraform-provider-datadog/blob/master/datadog/tests/data_source_datadog_roles_test.go#L31-L54

This will ensure that data source is working properly.

Reply:
I don't think it's needed, we already have a set of default rules that are created by default when the product is activated and I verified these are retrieved when they are read using the datasource, so the test is already not reading an empty list.